src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/organization_policy.py

```diff
@@ -20,7 +20,7 @@ def __init__(self):
     @staticmethod
     def _find_all(policy):
         _files = list(glob.iglob(
-            './adf-bootstrap/**/{0}.json'.format(policy),
```

Comment:
Could you rename this function to _find_all_paths, to clarify what it will return.
As this is part of the `OrganizationPolicy` class, one could otherwise think it is returning instances of that class.

Reply:
Good idea. Fixed.